generator-torsion/.gitignore

```diff
@@ -1,5 +1 @@
 node_modules
```

Comment:
Yes, if someone is going to work on the yeoman generator, it will be installing a bunch of dependencies through npm, but these should not be tracked in the repo